packs/journals/remaster-changes.json

```diff
@@ -0,0 +1,30 @@
d into Stone</em> has become <em>@UUID[Compendium.pf2e.spells-srd.CuziTxu5CyYbOt1L]{One with Stone}</em></li><li><em>Gentle Repose</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Peaceful Rest]</em></li><li><em>Flesh to Stone</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Petrify]</em></li><li><em>Dimensional Lock</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Planar Seal]</em></li><li><em>Magic Fang</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Runic Body]</em></li><li><em>Magic Weapon</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Runic Weapon]</em></li><li><em>See Invisibility</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.See the Unseen]</em></li><li><em>Longstrider</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Tailwind]</em></li><li><em>Tanglefoot</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</em></li><li><em>Mage Hand</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Telekinetic Hand]</em></li><li><em>Dimension Door</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Translocate]</em></li><li><em>Tongues</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Truespeech]</em></li><li><em>Gaseous Form</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Vapor Form]</em></li></ul><p>Some spells have had their mechanics or text updated with no change to the name:</p><ul><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Augury]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Detect Magic]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Fire Shield]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Invoke Spirits]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Speak with Animals]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Speak with Plants]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Speak with Stones]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Wish]</em></li></ul><p>You can also find the following new remaster spells in our compendiums. These are either entirely new spells, or new takes on old spells that have mechanical changes that are not clearly intended as a complete replacement, or the equivalent spell was kept in the system for other mechanical reasons.</p><ul><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Figment]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Frostbite]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.xxWhyl81w3ckslAU]{Howling Blizzard}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.6DfLZBl8wKIV03Iq]{Ignition}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.AuIiqc7jjiy1GZ75]{Manifestation}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.zgIlFMtzRkKfSay1]{Mirage}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.DCBwfPlPVrUvAOYJ]{Revealing Light}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.zDJS8E66UI0himqV]{Thunderstrike}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.yLJROsQtyrPIKcDx]{Wrathful Storm}</em></li></ul>",
```

Comment:
```suggestion
                "content": "<p>You'll notice some changes to the pathfinder system with this update. With the release of Rage of Elements we also received our first preview of the Pathfinder remastered rules. These changes are mostly in how things are named, with no mechanical impact. For example, you'll notice that the \"Flat-Footed\" condition is now renamed to \"Off-Guard\". You can read more of these changes in the <a href=\"https://paizo.com/pathfinder/corepreview\">Pathfinder Core Preview Document</a>. If you are unaware of the motivation for these changes, it was prompted by the need to remove terminology from the game used by a different company's game system.</p><p>We system developers want to make moving to the remastered version as smooth as possible for users, and for most groups these changes will have no impact. The new rules are compatible with the old rules, and all outward appearances are that this is mostly a renaming of items. The mechanical changes will not be integrated into the system until we have guidance on how to implement them with no interruption to gameplay. Going forward we plan to follow direction from Paizo and treat the remastered rules as errata to the game. We do not plan on removing any NPCs from the system, though in the future some names may change. There is no need to worry about something simply vanishing entirely on you or your group. We're players and GMs too, and we understand that some groups may be averse to changes, even ones that are just terminology changes.</p><p>We have added the new spells found in the preview document, including renaming some existing spells. If the mechanics of the spell changed in a way that it was not clear if it was meant as a replacement for an existing spell, then we kept the original spell as well as the spell intended to fill a similar function. For example, the <em>Ignition</em> spell is a new fire cantrip but because the mechanics of that cantrip are different than <em>Produce Flame</em> and the Psychic class interacts with the mechanics of produce flame directly, both spells have been kept in the system. But the new spell <em>Falling Stars</em> is a direct mechanical replacement for <em>Meteor Swarm</em>. Compendium entries of <em>Meteor Swarm</em> were renamed, and the mechanics of that spell updated to <em>Falling Stars</em>. Existing links to <em>Meteor Swarm</em> will now point to <em>Falling Stars</em>, but copies of the spell on actors were not removed or altered. For ease of reference here is a list of all spells that have had their names changed:</p><ul><li><em>Scorching Ray</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Blazing Bolt]</em></li><li><em>Burning Hands</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Breathe Fire]</em></li><li><em>Calm Emotions</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Calm]</em></li><li><em>Purify Food and Drink</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Cleanse Cuisine]</em></li><li><em>Entangle</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Entangling Flora]</em></li><li><em>Endure Elements</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Environmental Endurance]</em></li><li><em>Meteor Swarm</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Falling Stars]</em></li><li><em>Plane Shift</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Interplanar Teleport]</em></li><li><em>Know Direction</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Know the Way]</em></li><li><em>Stoneskin</em> has become <em>@UUID[Compendium.pf2e.spells-srd.2BV2yYPfVJ5zirZt]{Mountain Resilience}</em></li><li><em>Mage Armor</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Mystic Armor]</em></li><li><em>Tree Stride</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Nature's Pathway]</em></li><li><em>Barkskin</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Oaken Resilience]</em></li><li><em>Tree Shape</em> has become <em>@UUID[Compendium.pf2e.spells-srd.CJJ27kJew3nq0Du6]{One with Plants}</em></li><li><em>Meld into Stone</em> has become <em>@UUID[Compendium.pf2e.spells-srd.CuziTxu5CyYbOt1L]{One with Stone}</em></li><li><em>Gentle Repose</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Peaceful Rest]</em></li><li><em>Flesh to Stone</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Petrify]</em></li><li><em>Dimensional Lock</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Planar Seal]</em></li><li><em>Magic Fang</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Runic Body]</em></li><li><em>Magic Weapon</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Runic Weapon]</em></li><li><em>See Invisibility</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.See the Unseen]</em></li><li><em>Longstrider</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Tailwind]</em></li><li><em>Tanglefoot</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</em></li><li><em>Mage Hand</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Telekinetic Hand]</em></li><li><em>Dimension Door</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Translocate]</em></li><li><em>Tongues</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Truespeech]</em></li><li><em>Gaseous Form</em> has become <em>@UUID[Compendium.pf2e.spells-srd.Item.Vapor Form]</em></li></ul><p>Some spells have had their mechanics or text updated with no change to the name:</p><ul><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Augury]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Detect Magic]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Fire Shield]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Invoke Spirits]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Speak with Animals]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Speak with Plants]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Speak with Stones]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Wish]</em></li></ul><p>You can also find the following new remaster spells in our compendiums. These are either entirely new spells, or new takes on old spells that have mechanical changes that are not clearly intended as a complete replacement, or the equivalent spell was kept in the system for other mechanical reasons.</p><ul><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Figment]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.Item.Frostbite]</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.xxWhyl81w3ckslAU]{Howling Blizzard}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.6DfLZBl8wKIV03Iq]{Ignition}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.AuIiqc7jjiy1GZ75]{Manifestation}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.zgIlFMtzRkKfSay1]{Mirage}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.DCBwfPlPVrUvAOYJ]{Revealing Light}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.zDJS8E66UI0himqV]{Thunderstrike}</em></li><li><em>@UUID[Compendium.pf2e.spells-srd.yLJROsQtyrPIKcDx]{Wrathful Storm}</em></li></ul>",
```

Reply:
("Us system developers" --> "We system developers")

Reply:
grammar is silly, because while correct it sounds clunky.

Reply:
I find "us" clunky :)

> Us want to make moving to the remastered version as smooth as possible for users, and for most groups these changes will have no impact.

> > We want to make moving to the remastered version as smooth as possible for users, and for most groups these changes will have no impact.

Reply:
It does add some folksy charm to the US Constitution preamble, though!
> Us the People of the United States, in Order to form a more perfect Union, establish Justice, insure domestic Tranquility, provide for the common defence, promote the general Welfare, and secure the Blessings of Liberty to ourselves and our Posterity, do ordain and establish this Constitution for the United States of America.

Reply:
Those are different contexts. I agree that 'we' is grammatically correct, it just sounds worse to say "We system developers" vs "Us system developers". We can just say "We" and drop the system developers bit altogether and that sounds fine.